common/src/main/java/tachyon/util/UnderFileSystemUtils.java

```diff
@@ -29,8 +29,11 @@
   private UnderFileSystemUtils() {}
 
   /**
-   * Deletes the directory at the given path. If delete is unsuccessful, then this operation will
-   * throw a {@link java.io.IOException}.
+   * Deletes the directory at the given path.
```

Comment:
`in the under filesystem`, it may be confusing whether the directory is in Tachyon, or in local disk, or in ufs.